### DISCUSSION TITLE
How does the CoqGym project respect the licensing of the original Coq projects when generating data and doing experiments?

### DISCUSSION THREAD

**Discussion by brando90** [princeton-vl/CoqGym#75] [Q&A]
I am no license expert but my understanding is the GPL license is more restrictive than MIT ones. So my question is, how does CoqGym respect the licenses that the original Coq projects have when:

1. Generating data from them
2. Using the generated data for experiments

Thanks in advance!

**Comment by yangky11** [comment]
See https://github.com/princeton-vl/CoqGym/discussions/59.
